KeeperSdk/enterprise/EnterpriseData.cs

```diff
@@ -283,6 +285,137 @@ public async Task PopulateEnterprise()
                             }
                             break;
 
+                        case EnterpriseDataEntity.Roles:
```

Comment:
Please do not check in this pull request.
I am refactoring this area to make enterprise management modular.